pkgs/os-specific/linux/guvcview/default.nix

```diff
@@ -13,19 +13,23 @@ stdenv.mkDerivation rec {
     sha256 = "11byyfpkcik7wvf2qic77zjamfr2rhji97dpj1gy2fg1bvpiqf4m";
   };
 
-  buildInputs =
-    [ SDL2
-      alsaLib
-      ffmpeg
-      gtk3
-      intltool
-      libusb1
-      libv4l
-      pkgconfig
-      portaudio
-      udev
-      gsl
-    ] ++ stdenv.lib.optional pulseaudioSupport libpulseaudio;
+  nativeBuildInputs = [
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    SDL2
+    alsaLib
+    ffmpeg
+    gtk3
+    intltool
+    libusb1
+    libv4l
+    pkgconfig
+    portaudio
+    udev
+    gsl
+  ] ++ stdenv.lib.optional pulseaudioSupport libpulseaudio;
```

Comment:
while at this, can you move `intltool` and `pkg-config` to `nativeBuildInputs` ?

Reply:
Good idea.